share/instruments/instruments.xml

```diff
@@ -8691,7 +8690,6 @@
                   </Channel>
                   <genre>jazz</genre>
             </Instrument>
-            <!-- Keyboard Synth has been temporarily removed to make way for a new Synth Instrumental Group -->
```

Comment:
Comment removed intentionally. It has been recorded in a [separate spreedsheet tab](https://docs.google.com/spreadsheets/d/1SwqZb8lq5rfv5regPSA10drWjUAoi65EuMoYtG-4k5s/edit#gid=1466210644) for now.